runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/magic/Rs2Magic.java

```diff
@@ -70,7 +68,7 @@ public static void cast(MagicAction magicSpell) {
         if (magicSpell.getWidgetId() == -1)
             throw new NotImplementedException("This spell has not been configured yet in the MagicAction.java class");
 
-        Microbot.doInvoke(new NewMenuEntry(-1, magicSpell.getWidgetId(), menuAction.getId(), identifier, -1, "<col=00ff00>" + magicSpell.getName() + "</col>"), new Rectangle(0, 0, Microbot.getClient().getCanvasWidth(), Microbot.getClient().getCanvasHeight()));
+        Microbot.doInvoke(new NewMenuEntry("cast", -1, magicSpell.getWidgetId(), menuAction.getId(), identifier, -1, magicSpell.getName()), new Rectangle(1, 1));
```

Comment:
I guess natural mouse movement is not used for magic spells?

Reply:
Oversight by me, i have updated the code so that the natural mouse can be used with magic spells